test_files/file_comment.puretransform/comment_before_var.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * @fileoverview This comment must not be emitted twice.
+ * @mods {google3.java.com.google.javascript.typescript.examples.boqui.boqui}
+ * @modName {foobar}
```

Comment:
Is mods and modName load bearing here?  Either explain, rename test, or generate 2 tests - one for `@mods` and one for comment before a constant.

Reply:
This is a 1:1 copy of the previously existing test (`file_comment/comment_before_var.ts`). The original test makes sure that merging a file overview comment before an exported var works. It must contain markers that it is a fileoverview comment. It does not quite matter what those are (fileoverview or mods etc), but this repros the exact problem we had in google3.

In the puretransform case, the tags in the comment are insignificant, but we generally try to keep the files identical between multiple variants (read: copies) of the same test case for different situations.

I could either change both or keep both this way I think. Given that changing them doesn't fix anything and is unrelated to this PR, I think I'd rather keep them the way they are.